### ISSUE TITLE
illegal base64 data at input byte 48

### ISSUE THREAD

**Issue by zeminzhou** [VictoriaMetrics/VictoriaMetrics#4692]
### Describe the bug

VictoriaMetrics can't decode some request sent by prometheus client_golang.

### To Reproduce

1. send metrics to `victoria-metrics-pure` by prometheus client_golang
```
  labels := map[string]string{}
  labels["region"] = "az-southeast-1-f01/wzy-az-southeast-1"
  addr := "http://127.0.0.1:8428/api/v1/import/prometheus"
  pusher := push.New(addr, job).Format(expfmt.FmtText).Gatherer(prometheus.DefaultGatherer)
  for k, v := range labels {
    pusher = pusher.Grouping(k, v)
  }
  for {
    select {
       case <-ctx.Done():
          return
       case <-time.After(interval):
              pusher.Push()
     }
  }
```
2. start `victoria-metrics-pure` server
3. encounter err ` illegal base64 data at input byte 48`

### Version

victoria-metrics-20230721-115344-heads-master-0-g866b150f0

### Additional information

For code, prometheus client_golang encodes using [RawURLEncoding](https://github.com/prometheus/client_golang/blob/644c80d1360fb1409a3fe8dfc5bad4228f282f3b/prometheus/push/push.go#L353), but VictoriaMetrics decodes using [URLEncoding](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/866b150f0f5bc19c4ce658247d8c39d85635ed16/lib/protoparser/common/extra_labels.go#L67)

**Comment by valyala** [comment]
@zeminzhou , the issue should be fixed in the commit 7e5555f9c7b17653531d52ca44ee6819c7f1a6fa , which will be included in the next release. In the mean time it is possible to verify the bugfix by building VictoriaMetrics from this commit according to [these instructions](https://docs.victoriametrics.com/#how-to-build-from-sources) and then checking whether the issue is fixed there.

**Comment by valyala** [comment]
The issue should be fixed in [VictoriaMetrics v1.92.0](https://docs.victoriametrics.com/CHANGELOG.html). Closing it as fixed then.

**Comment by z2665** [comment]
> The issue should be fixed in [VictoriaMetrics v1.92.0](https://docs.victoriametrics.com/CHANGELOG.html). Closing it as fixed then.

Thanks for all your hard work. Will this bug fix be updated to the lts version?

**Comment by zeminzhou** [comment]
> The issue should be fixed in [VictoriaMetrics v1.92.0](https://docs.victoriametrics.com/CHANGELOG.html). Closing it as fixed then.

Can cherry-pick to v1.86?

**Comment by valyala** [comment]
FYI, the bugfix for this issue has been backported to [long-term support release v1.87.7](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.87.7).
